apisix/plugins/proxy-cache/memory_handler.lua

```diff
@@ -213,11 +213,19 @@ function _M.access(conf, ctx)
     end
 
     if err then
-        core.response.set_header("Apisix-Cache-Status", "MISS")
-        if err ~= "not found" then
+        if err == "expired" then
+            core.response.set_header("Apisix-Cache-Status", "EXPIRED")
+
```

Comment:
Can you remove these empty lines?

Reply:
I decided to write this after seeing the example of CODE_STYLE. 
[blank-line](https://github.com/apache/apisix/blob/master/CODE_STYLE.md#blank-line)